File: features/support/pages/janus/landing.rb
require_relative File.join(Rails.root, 'features', 'support', 'page.rb').to_s

module JanusHelper
  class Landing < Page
    def self.visit
      page.visit JanusNavigation::CONFIG_MAPPING['apps']['landing page']
      new
    end
  end
end
World(JanusHelper)

File: features/support/pages/javascript/janus/landing.js
/**
 * Created by scalixte on 4/30/15.
 */
'use strict';

var information = information || require('../../../../support/javascript/information');
var support = support || require('../../../../support/javascript/support');

var Landing = function(){
    this.config = information.config['apps'];
    this.selector = information.selector;
};

Landing.prototype.visit = function (that, callback) {
    support.get(that, this.config['landing page'], function(){
        setTimeout(callback, 1000);
    });
};

Landing.prototype.selectStudy = function (that, study, callback) {
    console.log('helllo');
};

module.exports = new Landing();
